data/json/vitrification/overmap_vitrified.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "quiet_farm",
+    "locations": [ "forest" ],
+    "overmaps": [
+      { "point": [ 0, -1, 0 ], "overmap": "dirt_road_forest_north" },
+      { "point": [ 0, -2, 0 ], "overmap": "dirt_road_turn_forest_west" },
+      { "point": [ 1, -2, 0 ], "overmap": "dirt_road_forest_west" },
+      { "point": [ 2, -2, 0 ], "overmap": "dirt_road_forest_west" },
+      { "point": [ 3, -2, 0 ], "overmap": "dirt_road_forest_west" },
+      { "point": [ 0, 0, 0 ], "overmap": "unvitrified_farm_0" },
+      { "point": [ 0, 0, 1 ], "overmap": "unvitrified_farm_1" },
+      { "point": [ 0, 0, -1 ], "overmap": "unvitrified_farm_-1" },
+      { "point": [ 0, 0, 2 ], "overmap": "unvitrified_farm_2" },
+      { "point": [ 1, 0, 0 ], "overmap": "unvitrified_orchard" }
+    ],
+    "connections": [ { "point": [ 4, -2, 0 ], "terrain": "road", "connection": "local_road", "from": [ 3, -2, 0 ] } ],
```

Comment:
```suggestion
    "connections": [ { "point": [ 4, -2, 0 ], "terrain": "road", "connection": "local_road", "existing": true, "from": [ 3, -2, 0 ] } ],
```
It doesn't make much sense for it to spawn it's own regular road when it's already got the dirt road, you might just need to up the occurrences a lil bit to compensate it being more fussy
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/45432782/3a7b9015-e2b9-4a59-8d74-630e6a397935)